keps/sig-release/3027-slsa-compliance/README.md

```diff
@@ -0,0 +1,894 @@
+-->
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [ ] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [ ] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input (including test refactors)
+  - [ ] e2e Tests for all Beta API Operations (endpoints)
+  - [ ] (R) Ensure GA e2e tests for meet requirements for [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+  - [ ] (R) Minimum Two Week Window for GA e2e tests to prove flake free
+- [ ] (R) Graduation criteria is in place
+  - [ ] (R) [all GA Endpoints](https://github.com/kubernetes/community/pull/1806) must be hit by [Conformance Tests](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md) 
+- [ ] (R) Production readiness review completed
+- [ ] (R) Production readiness review approved
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+<!--
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+-->
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+This document proposes a plan to harden the Kubernetes releases by
+making the necessary adjustments to comply with the SLSA Framework.
+[SLSA (Supply-chain Levels for Software Artifacts)](https://slsa.dev/)
+is a framework to harden software supply currently being defined by the 
+[OpenSSF](https://openssf.org/)'s 
+[Supply Chain Integrity WG](https://github.com/ossf/wg-supply-chain-integrity).
+
+The framework provides requirements and recommendations to software
+build systems to harden their environments and the processes that drive
+them. It also defines the metadata that needs to be produced to trace the 
+origins of every item in a software release.
+
+The main goal of this enhancement is to provide downstream consumers of our
+artifacts the highest assurance about the integrity of each Kubernetes release.
+
+SLSA defines several levels of hardening, each touching more aspects of the
+release process that go beyond its technical implementation. This document is
+meant to serve as a guide to reach the highest possible levels after 
+consensus has been reached about their viability.
+
+<!--
```

Comment:
I would additionally state that we're not only proposing SLSA for k/k artifacts, but also as a standard recommendation (no must) for every other project in the kubernetes and kubernetes-sigs org.

Reply:
.. added a line around this. Recommending and also sharing tools and practices as we learn them :+1: